features/step_definitions/login_steps.rb

```diff
@@ -0,0 +1,60 @@
+Given("I have registered an account") do
+  User.create(:username => "Bono", :password => "password")
+end
+
+When("I login with a registered username and password") do
+  visit login_path
+  fill_in 'username', with: "Bono"
+  fill_in 'password', with: "password"
+  click_button("Login")
```

Comment:
Good for now. Suggest we extract login into a method and/or create a more general matcher and call it from these specific matchers.